### PR TITLE
Coldfront publication additions

### PR DESCRIPTION
**Pull request by KaninB**
Additions for the two publication summary pages

- [ ] @matthew-kusz

### CODE REVIEW

File: coldfront/components/site/templates/urls.py
Comment (matthew-kusz) [inline coldfront/components/site/templates/urls.py:1]: Why was this file added here? The python files should not be in the templates folder.

File: coldfront/components/site/templates/views.py
Comment (matthew-kusz) [inline coldfront/components/site/templates/views.py:1]: Same thing for this file being in the templates folder.

File: coldfront/config/plugins/cas_login.py [hunk 1]
```diff
@@ -5,6 +5,9 @@
     'django_cas_ng',
 ]
 
+INSTALLED_APPS += [
+    'django_extensions'
+]
```
Comment (matthew-kusz) [inline coldfront/config/plugins/cas_login.py:10]: This is only needed for local testing and doesn't need to be in the main repo.

File: coldfront/core/portal/views.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
 from coldfront.core.research_output.models import ResearchOutput
 from coldfront.core.utils.common import import_from_settings
 
+import os
```
Comment (matthew-kusz) [inline coldfront/core/portal/views.py:25]: Extra import?

File: coldfront/templates/common/authorized_navbar.html
Comment (matthew-kusz) [inline coldfront/templates/common/authorized_navbar.html:1]: This file shouldn't be changed since we are using our own.

File: coldfront/templates/common/nonauthorized_navbar.html
Comment (matthew-kusz) [inline coldfront/templates/common/nonauthorized_navbar.html:1]: Same with this one.

File: coldfront/components/site/templates/urls.py
Comment (KaninB) [reply coldfront/components/site/templates/urls.py:1]: Unsure, I may have moved it on accident. I'm fine with removing it if you'd like.

**Comment by matthew-kusz** [comment]
Everything looks good, thanks!
